docs/getting-started/reference/glossary.md

```diff
@@ -0,0 +1,119 @@
+### Light Node
+
+A light node is a [resource-limited](#resource-limited) device or client that selectively subscribes to specific [messages](#waku-message) transmitted by other [peers](#peer) on the network.
+
+### Light Push
+
+Light push is a protocol enabling [light nodes](#light-node) to send [messages](#waku-message) to the [Relay](#relay) network and receive acknowledgments confirming that a [peer](#peer) has received them.
+
+### Node
+
+A node is a device or client that plays an active role in the operation of the [Waku network](#waku), participating to enable secure and private peer-to-peer Web3 communication.
+
+### Noise
+
+Noise is the integration of the Noise Protocol Framework into the [Waku network](#waku) to enable secure key-exchange mechanisms for encrypted communication with confidentiality, authenticity, integrity, strong forward secrecy, and identity-hiding properties.
+
+### Payload
+
+Payload provides guidelines for implementing secure and private communication in the network. It covers encryption, decryption, and signing methods for message payloads, focusing on confidentiality, authenticity, integrity, and unlinkability.
+
+### Peer
+
+A peer refers to other [nodes](#node) and participants of the [Waku network](#waku) with whom communication and interaction are possible.
+
+### Peer Discovery
+
+Peer discovery is the process where a [node](#node) locates and connects with [peers](#peer) to establish communication and exchange information.
+
+### Peer Exchange
+
+Peer exchange is a [peer discovery](#peer-discovery) mechanism that enables [light nodes](#light-node) to request and receive peers from other nodes in the network, allowing them to bootstrap and expand their connections without depending on [Discv5](#discv5).
+
+### Protocol
+
+A protocol is a set of rules that enables [nodes](#node) within the [Waku network](#waku) to perform various functionalities such as message sending, relaying, filtering, storing, retrieving, and more.
+
+### Pub/Sub
+
+Publish/Subscribe (Pub/Sub) is a messaging pattern where publishers distribute messages to subscribers without the direct knowledge of recipients, allowing efficient one-to-many communication.
+
+### Pub/Sub Topic
+
+A Pub/Sub topic is a routing mechanism for messages, indicating a subscription to receive messages on that topic for relay purposes.
+
+### Rate Limit Nullifiers
+
+Rate Limit Nullifiers (RLN) are a construct based on zero-knowledge proofs that provide an anonymous rate-limited messaging framework, preserving message owner anonymity while preventing spam or DoS attacks.
+
+### Relay
+
+Relay is a [protocol](#protocol) that extends the [GossipSub protocol](#gossipsub) to enable secure and censorship-resistant [message](#waku-message) dissemination among [peers](#peer) while preserving privacy.
+
+### Resource-Limited
+
+Resource-limited refers to environments or devices restricting available resources, including bandwidth, CPU, memory, disk, and battery power.
```

Comment:
add "mostly offline" in the keywords and maybe good to add example of browser and mobile phone.